public/javascripts/SVLabel/src/SVLabel/label/Label.js

```diff
@@ -423,6 +423,17 @@ function Label (svl, pathIn, params) {
             // Renders the label image.
             path.render2(ctx, pov);
 
+            //Draws label outline
```

Comment:
Can you modify the comment slightly to fit the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)?